### DISCUSSION TITLE
How to customize the order of sidebar items?

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [fuma-nama/fumadocs#585] [Q&A] [answered]
In the last version, it is not clear how to customize the order of sidebar items, any tips are appreciated!

**Comment by olabrahmi** [comment, marked as answer]
Use `meta.json` file. Please check examples in the fumadocs repo for more info.

**Comment by fuma-nama** [comment]
Guess I'm not clear enough, maybe I can rename this to "Organising Pages"?

![image](https://github.com/fuma-nama/fumadocs/assets/76240755/ea368f38-fde0-4a29-90ed-f0dc42d246ac)

**Reply by Ali-Hussein-dev** [reply]
Idk where the screenshot comes from. 

When I was searching for "sidebar", I found nothing, I navigated to core/sidebar and also found nothing related to customizing the order of the sidebar items.

My 2 cents suggestion is that the sidebar settings should be organized in one place, or have all links for adjusting the sidebar in one section. This would make it easier for developers to quickly access and understand the settings without getting confused or having to learn new abstractions before finding what they are looking for.

**Reply by fuma-nama** [reply]
This is from the Getting Started section of Fumadocs UI, because in Fumadocs we use Page Tree for navigation (the same order applies for breadcrumbs, footer and other navigation elements) so sidebar honestly doesn't sound like a good name. I'll add this to the FAQ section
